scripts/pop_demix.py

```diff
@@ -1,6 +1,23 @@
 from neurophotonics.demix import *
 
-IlluminationCycle.populate(reserve_jobs=True, processes=1024, display_progress=False)
-Demix.populate('sample<=5', display_progress=False, processes=1024, reserve_jobs=True)
-Cosine.populate(reserve_jobs=True, display_progress=False, processes=1024)
-SpikeSNR.populate(reserve_jobs=True, display_progress=False, processes=1024)
+try:
+    IlluminationCycle.populate(reserve_jobs=True, processes=1024, display_progress=False)
```

Comment:
1024 seems like a very larrge number of processes.

Reply:
Python has code to set the number of processes equal to the minimum of this value, the number of keys left to run, and the number of processors available. In this code, processes isn't the number of processes being created, but an upper bound on how many can be spawned.

Since we want to use as many processors as we can, this value should be set to a number that will always be greater than the number of processors the machine running the code has available, regardless of which machine that may be.

Reply:
The main reason for this is taken into account this [snippet](https://github.com/datajoint/datajoint-python/blob/56ced5dabe84687ce33f14c02889fe546c9bbc80/datajoint/autopopulate.py#L215-L216) from DJ Python. We are just trying to ensure that we use the maximum cores available on the system since `mp.cpu_count()` will determine the available cores.

```python
        if processes > 1:
            processes = min(processes, nkeys, mp.cpu_count())
```

Reply:
This needs to be fixed in `datajoint-python`. Shouldn't throw an error.